test.sh

```diff
@@ -1 +1 @@
-java -cp build/user-agent-detector.jar net.pieroxy.ua.tooling.UserAgentTester test test-data/database.gz
+java -cp target/user_agent_detector-2015-10-31-SNAPSHOT.jar net.pieroxy.ua.tooling.UserAgentTester test src/test/resources/database.gz
```

Comment:
What does the timestamp mean there? Will this be stable over time ?

Sorry about the stupid questions, but I've managed my way through the Java world without approaching maven so I'm quite the n00b when it comes to that.